### ISSUE TITLE
NTR: Abdominal imaging: gastric rupture (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8989]
New term request Abdominal imaging: gastric rupture (elthat_110514023448): Stomach; abdomen  UMLS: 0038357

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000383
- label: Gastric perforation
- parents: [Abnormal stomach morphology (HP:0002577)](https://hpo.jax.org/app/browse/term/HP:0002577)
- pmid: PMID:30137838
- def: A full-thickness injury of the wall of the stomach.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8989
- synonyms: Gastric rupture(NONE)
